test/server/graphql/v2/mutation/AddFundsMutations.test.js

```diff
@@ -118,6 +136,26 @@ describe('server/graphql/v2/mutation/AddFundsMutations', () => {
       expect(result.data.addFunds.amount.currency).to.equal('USD');
     });
 
+    it('can add funds as host admin with authorization', async () => {
+      const userToken = await fakeUserToken({ scope: ['host'] });
+      const result = await oAuthGraphqlQueryV2(
+        addFundsMutation,
+        {
+          account: { legacyId: collective.id },
+          fromAccount: { legacyId: randomUser.CollectiveId },
+          amount: { value: 20, currency: 'USD', valueInCents: 2000 },
+          description: 'add funds as admin',
+          hostFeePercent: 6,
+        },
+        userToken,
+        hostAdmin,
```

Comment:
It's wrong to have a user token not match the user we're using it for; in fact, that can't happen in the context of real queries as resolved by `_authenticateUserByJwt`.